src/main/java/org/folio/inventory/support/http/client/OkapiHttpClient.java

```diff
@@ -34,12 +35,21 @@
   private final Consumer<Throwable> exceptionHandler;
 
   public OkapiHttpClient(HttpClient httpClient,
-                         URL okapiUrl,
-                         String tenantId,
-                         String token,
-                         String userId,
-                         String requestId,
-                         Consumer<Throwable> exceptionHandler) {
+    WebContext context, Consumer<Throwable> exceptionHandler)
+    throws MalformedURLException {
+
+    this(httpClient, new URL(context.getOkapiLocation()),
+      context.getTenantId(), context.getToken(), context.getUserId(),
+      context.getRequestId(), exceptionHandler);
+  }
+
+  public OkapiHttpClient(HttpClient httpClient,
+    URL okapiUrl,
+    String tenantId,
+    String token,
```

Comment:
Can you add javadoc that null, an empty ("") String or a whitespace only means no token header?